core/src/main/resources/jenkins/model/Jenkins/projectRelationship.properties

```diff
@@ -0,0 +1,27 @@
+# The MIT License
+#
+# Bulgarian translation: Copyright (c) 2024, Tim Jacomb
```

Comment:
Seems out of place, this is the English file?

Reply:
whyyyy is that there, I just copied the header from that file the only one with that -.-